src/views/search_criteria/SearchCriteriaView.js

```diff
@@ -39,14 +39,8 @@ class SearchCriteriaView extends InputViewBase {
         this.bindEvents(this.mediator);
     }
 
-    bindEvents(mediator) {
+    bindEvents(_mediator) {
```

Comment:
The `search:example` event is no longer used anywhere.  I could probably just remove the parameter for `_mediator`, or just move the single remaining callback to the initialize method; leaving it like this might make it easier for if we add more events later, though.